src/components/Cards/CreateWiki.vue

```diff
@@ -228,6 +228,10 @@ export default {
         this.hasError = true
         this.error.terms = errors.terms[0]
       }
+      if (errors.tooManyWikis) {
+        this.hasError = true
+        this.displayGenericError(errors.message)
```

Comment:
This obviously works fine but it "feels" like we would maybe want to TODO: display `errors.message` in almost all cases